inst/tinytest/testPlot.R

```diff
@@ -879,6 +904,37 @@ expect_identical_xl(
   c("feature_0006", "feature_0002")
 )
 
+# noMapping (multiModel) -------------------------------------------------------
+
+# testStudyObjNoMapping <- testStudyObj
+# testStudyObjNoMapping[["mapping"]] <- list()
+# suppressMessages(installStudy(testStudyObjNoMapping))
```

Comment:
You don't need to install the study. `plotStudy()` can work directly on the onStudy object. Please remove this call to `installStudy()` and the one below. Also, please uncomment these tests so that they run on GitHub Actions.